docs/source/tutorials/parsl-rp.ipynb

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
<p>I am not sure this comment is useful. Looking at the config file, we have no explicit parameter for either a function or executable task. We should instead explain that: (1) we use the master/worker paradigm, and (2) some of their config parameters, e.g., ranks, worker_class and worker_file. (note how we should NOT use the word 'master' especially in a tutorial).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ExaWorks/SDK/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>